tools/ci-build.sh

```diff
@@ -46,15 +46,18 @@ export P4C_PIP_PACKAGES="ipaddr \
                           ply==3.8 \
                           scapy==2.4.5"
 
-
-
 apt-get update
 apt-get install -y --no-install-recommends \
   ${P4C_DEPS} \
   ${P4C_EBPF_DEPS} \
   ${P4C_RUNTIME_DEPS} \
   git
 
+# TODO: Remove this rm -rf line once the ccache memcache config is removed.
+rm -rf /usr/local/etc/ccache.conf
```

Comment:
Is the `rm -rf` line intended to be removed before merging this pull request?

Reply:
No, we can only remove it once we have removed the corresponding ccache code in https://github.com/p4lang/third-party.